src/utils/stateReducers/pagination.js

```diff
@@ -0,0 +1,33 @@
+const INITIAL_PAGINATION_STATE = {
+  page: 1,
+  start: 0,
+  query: ''
+};
+
+const paginationReducer = (state, action) => {
```

Comment:
Any objection to moving this to `src/core/redux/reducers`?

Reply:
This is not a redux reducer. It's a reducer used for local state.